Prototypes/test/nye_Bee.pde

```diff
@@ -0,0 +1,180 @@
+int Xbee = 950;
+int Ybee = 500;
+int speed = -5;
+float bbr = 245; //Body Color R
+float bbg = 229; //Body Color G
+float bbb = 47;  //Body Color B
+
+float bgr = 136; //Background color G
+float bgg = 160; //Background color R
+float bgb = 245; //Background color b
+ 
+  int bbws = 0;     //Bee Back Wings stroke
+  int bbwf = 200;   //Bee Back Wing fill
+  int bbwsw = 1;    //Bee Back WIng StrokeWeight
+  int bbwcx = 15;   //Bee Back Wing Center X
+  int bbwcy = -30;  //Bee Back Wing Center Y
+  int bbwh = 40;    //Bee Back Wing Height
+  int bbwl = 60;    //Bee Back Wing Length
+  
+  int bblsw = 3;    //Bee Back Leg StrokeWeight
+  int bbl1x1 = 0;   //First Bee Back Leg x1
+  int bbl1y1 = 0;   //First Bee Back Leg y1
+  int bbl1x2 = -10; //First Bee Back Leg x2
+  int bbl1y2 =  45; //First Bee Back Leg y2
+  
```

Comment:
I recommend to use the PVector class instead:
```
PVector firstLegPosition = new PVector(0, 0);
```
Even better, save the positions in an array:
```
// Defines an array of vector points
PVector[] legPositions = new PVector[] {
  new PVector(0, 0),
  new PVector(-10, 45)
};

// How to use:
legPositions[0] // =>new PVector(0, 0)
legPositions[0].x // => 0
legPositions[0].y // => 0

legPositions[1] // =>new PVector(-10, 45)
legPositions[1].x // => -10
legPositions[1].y // => 45
```

---

https://processing.org/reference/PVector.html